src/software/integration_tests/BUILD

```diff
@@ -0,0 +1,15 @@
+package(default_visibility = ["//visibility:public"])
+
+cc_test(
+    name = "integration_tests",
```

Comment:
Hmm, perhaps `simulated_test` might be a bit more descriptive? "integration tests" encompass more then just simulated ones.

Reply:
renamed all instances to `simulated_test`